geocolor/plugins/modules/algorithms/visir/GeoColor.py

```diff
@@ -96,8 +96,19 @@ def normalize_ir_by_abslats(ir, lats):
     abslats = np.ma.abs(lats)
     abslats[abslats < 30.0] = 30.0
     abslats[abslats > 60.0] = 60.0
+
+    # maxir = 300.0
+    # mnil = 210.0
+    # mnih = 230.0
+    # minir = mnil + 20.0 * (abslats - 30.0) / (60.0 - 30.0)
+    # # IDL GeoColor code sets min/max values to mnil/mnih
+    # minir[minir < mnil] = mnil
+    # minir[minir < mnih] = mnih
+    # normir = (ir - minir) / (maxir - minir)
```

Comment:
Are these just comments to describe what is here? Or is this commented out? Should it just be removed or do we expect to go back to it someday?